frontend/components/Teams/SeparateTeam.jsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+
+export default class SeparateTeam extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = { team: {} };
```

Comment:
if we decided to use hooks and functional components change this.state to useState hook